### DISCUSSION TITLE
Consider registering as Github sponsor

### DISCUSSION THREAD

**Discussion by bilfeldt** [Kyon147/laravel-shopify#4] [Ideas]
@Kyon147 Instead of the current Paypal sponsor link. Consider registering for Github sponsor as it is so much easier to start sponsoring there 👍  https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-personal-account

**Comment by Kyon147** [comment]
Not a bad idea, I'll take a look!

**Comment by Kyon147** [comment]
It's not perfect but something, https://github.com/sponsors/Kyon147

**Reply by bilfeldt** [reply]
Yes! Got to be the first sponsor 🥇 🥳

**Comment by talktohenryj** [comment]
@Kyon147 thanks so much for taking this over. I know I speak for many of us when I say this repo is very important to our overall existence as app owners. I am not in the position just yet to sponsor on a monthly basis but will be back shortly to give. What else can we do to help this repo continue to grow? \

**Reply by Kyon147** [reply]
Thanks for the kind words and any help is appreciated. Creating PRs for bugs, answering issues, updating the wiki etc. 

It all helps move the package forward.
